manubot/cite/util.py

```diff
@@ -267,12 +276,12 @@ def citation_to_citeproc(citation, prune=True):
 
     note_text = f'This CSL JSON Item was automatically generated by Manubot v{manubot_version} using citation-by-identifier.'
     note_dict = {
-        'standard_citation': citation,
```

Comment:
This will affect reading and writing CSL JSON files, right?  Is there any risk of introducing incompatibilities with existing CSL JSON?  It looks like the answer is no because `csl_item_set_standard_id` looks for `standard_id` and then `standard_citation`.

Reply:
Adding the note was recently added in https://github.com/manubot/manubot/pull/104, which has not been included in rootstock, so I don't think there will be any CSL JSON notes that use `standard_citation`. Also I don't think there are many cases where it would matter.